web/Dockerfile

```diff
@@ -2,13 +2,18 @@ FROM node:12-alpine as base
 ENV REACT_APP_APPLICATION_OWNER=jmgo@equinor.com
 ENV REACT_APP_CLIENT_ID=1dbc1e96-268d-41ad-894a-92a9fb85f954
 ENV REACT_APP_SCOPES="api://lost-circulation-material-api/Optimization.All.All"
+ARG UID=2000
+ARG GID=2000
 
+
+RUN addgroup -S usergroup -g $GID && adduser -u $UID -S nonrootuser -G usergroup
 WORKDIR /code
 ADD package.json yarn.lock /code/
 RUN yarn install --frozen-lockfile
 ADD ./ /code/
 
 FROM base as development
+USER nonrootuser 
 CMD ["yarn", "start"]
 
 FROM base as build
```

Comment:
I'm quite sure line 23 will fail here.
Have you tested building the "prod" stage of the container, not just the "development"?